lib/ui/screens/search/search_view.dart

```diff
@@ -34,7 +34,16 @@ class SearchView extends StatelessWidget {
               shrinkWrap: true,
               itemCount: model.filteredResults.length,
               itemBuilder: (BuildContext context, int index) => ListTile(
-                title: Text(model.filteredResults[index]),
+                title: Text(model.filteredResults[index].title),
+                subtitle: Text('${model.filteredResults[index].body.substring(
```

Comment:
**suggestion:** This can be done by using the the parameters of the text Widget.
```
                 ListView.builder(
              shrinkWrap: true,
              itemCount: model.filteredResults.length,
              itemBuilder: (BuildContext context, int index) => ListTile(
                title: Text(
                  model.filteredResults[index].title,
                  textAlign: TextAlign.left,
                  maxLines: 1,
                  overflow: TextOverflow.ellipsis,
                ),
                subtitle: Text(
                  model.filteredResults[index].body,
                  textAlign: TextAlign.left,
                  maxLines: 1,
                  overflow: TextOverflow.ellipsis,
                ),
                onTap: () => controller.openPost(
                  postId: model.filteredResults[index].id,
                ),
              ),
            ),
```